core/documents-manager.php

```diff
@@ -140,7 +144,7 @@ public function	ajax_save( $request ) {
 		$document = $this->get( $request['post_id'] );
 
 		if ( ! $document->is_built_with_elementor() || ! $document->is_editable_by_current_user() ) {
-			throw new \Exception( 'no_access' );
+			throw new \Exception( 'Access Denied' );
```

Comment:
See: https://github.com/pojome/elementor/pull/3282

Use: `throw new \Exception( 'Access denied.' );`

and update the phpDocs to: `@throw \Exception If current user don't have permissions to edit the post or the post is not using Elementor.`